plugins/python/pyloader.c

```diff
@@ -22,7 +22,7 @@ PyMethodDef uwsgi_eventfd_write_method[] = { {"uwsgi_eventfd_write", py_eventfd_
 void set_dyn_pyhome(char *home, uint16_t pyhome_len) {
 
 
-	char venv_version[15];
+	char venv_version[30];
```

Comment:
Shouldn't one more char be enough?

Reply:
Yes you could use `16` chars if you like. But what if one day there is a Python `10.402`? What is the gain of having *just-enough* bytes? If you prefer, you can use `16`. I left the right for the reviewer to make corrections.